src/components/pages/Navigation.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom';
+import NameForm from './StatedNameForm';
+import GraphForm from './StatedGraphForm';
 
 const Navigation = () => {
```

Comment:
Navigation now not only contains navigation bar, but also welcome message and routing.
Please divide stuff to components (or even remove them as navigation bar is not yet needed).
And rename it to something like Routing.tsx.